src/component/GjennomforingGenerellInfo.tsx

```diff
@@ -13,19 +14,28 @@ interface GjennomforingGenerellInfoProps {
 export const GjennomforingGenerellInfo = ({ gjennomforing, className }: GjennomforingGenerellInfoProps): React.ReactElement => {
 	const oppstart = formatDateMedMndNavn(gjennomforing.startDato)
 	const sluttdato = formatDateMedMndNavn(gjennomforing.sluttDato)
+	const avsluttet = gjennomforing.status === GjennomforingStatus.AVSLUTTET
 
 	const arrangor = gjennomforing.arrangor
 
 	return (
 		<div className={className}>
 			<div className={styles.rad}>
 				<BodyShort size="small" className={styles.sammendrag}>{gjennomforing.opprettetAr}/{gjennomforing.lopenr}</BodyShort>
-				<BodyShort size="small" className={styles.sammendrag}>{gjennomforing.tiltak.navn}</BodyShort>
+				{
+					avsluttet &&
```

Comment:
Kan hende det hadde vært cleanere å flytte denne ut til en annen komponent. Ser forsåvidt at samme merkelapp ligger lengre ned. Kunne for eksempel lagt til skalVises prop og sende med avsluttet=true/false

Reply:
Er det dette du tenker kan være en egen komponent? 
```
<div className={styles.tags}>
								<Tag variant="warning" size="small">
								Avsluttet
								</Tag>
							</div>
```

Reply:
ja, enten det eller et annet alternativ:
```
const AvsluttetMerkelapp = ({hidden}: {hidden: boolean} ) => {
     if(hidden) return null
     return  
          <div className={styles.tags}>
               <Tag variant="warning" size="small">
                   Avsluttet
              </Tag>
          </div>
}
```

Da abstraherer man bort og gjør parenten cleanere, men det er noen som ikke liker det patternet fordi det delegerer for mye ansvar til child komponenten. Personlig syns jeg lesbarhet er viktigst

Reply:
Det ble veldig mye mer ryddig. Og så slipper man å endre mange steder hvis man f.eks. ønsker en annen type tag eller noe :)

Reply:
måtte gjøre det litt annerledes fordi kompilatoren ble sur hvis ikke den fikk returnere et react element :/